games/common/components/StepperOptions.js

```diff
@@ -142,7 +156,10 @@ define([
 				const maxFrame = Math.max(Math.round(this.maxFrame.value), 1);
 				this.trigger('changegame', [{maxFrame}]);
 			});
-
+			this.skipFrame.addEventListener('change', () => {
+				const skipFrame = Math.max(Math.round(this.skipFrame.value), 0);
+				this.trigger('changegame', [{skipFrame}]);
```

Comment:
I don't like storing the skip frame as a game configuration state. It feels like a property of the **event**, not the **game**